### PR TITLE
SMUS PIDOR

### PR DESCRIPTION
**Pull request by hantarex08**
ti pizduk

### CODE REVIEW

File: homa228.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+print ('ax**2+bx+c=0')
+a = int(input())
```
Comment (pkovarsky) [inline homa228.py:2]: Add print message for each variable

Comment (hantarex08) [reply homa228.py:2]: in english?

Comment (pkovarsky) [reply homa228.py:2]: It would be good if you add validation for each variable. When I type string, I see exception message

**Comment by pkovarsky** [comment]
/

**Pull request closed without merging**